src/shogun/mathematics/linalg/backend/eigen/BasicOps.cpp

```diff
@@ -446,6 +477,63 @@ void LinalgBackendEigen::element_prod_impl(
 	result_eig = a_eig.array() * b_eig.array();
 }
 
+/* Helper method to compute elementwise division with Eigen */
```

Comment:
@gf712 just another illustration for the fact that we need a data structure for linear algebra. That can for example have a ".T" flag inside :)

Reply:
yup, like eigen.. Something being worked on in the graph stuff!

Reply:
:rocket: